smartlogic/handlers.go

```
@@ -146,5 +187,8 @@ func writeResponse(rw http.ResponseWriter, updateStatus status, err error) {
 
 func writeJSONError(w http.ResponseWriter, errorMsg string, statusCode int) {
 	w.WriteHeader(statusCode)
-	fmt.Fprintln(w, fmt.Sprintf("{\"message\": \"%s\"}", errorMsg))
+	_, err := fmt.Fprintln(w, fmt.Sprintf("{\"message\": \"%s\"}", errorMsg))
+	if err != nil {
+		return
```

This isn't doing anything meaningful. Better handling would be to ignore the error altogether: `_, _ = fmt.Fprintln(w, fmt.Sprintf("{\"message\": \"%s\"}", errorMsg))`